roles/pm2/manage/tasks/main.yml

```diff
@@ -23,6 +23,13 @@
     - restart
     - start
 
+- name: Generate and enable PM2 startup script
+  ansible.builtin.command:
+    cmd: sudo env PATH=$PATH:/usr/bin pm2 startup systemd -u {{ ansible_user }} --hp /home/{{ ansible_user }}
```

Comment:
As per our contribution guidelines, we want to use long version options for readability  :)

Reply:
I thought it was only for documentation